pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGenerator/index.jelly

```diff
@@ -56,10 +57,10 @@
                                     <j:set var="help" value="${descriptor.helpFile}"/>
                                     <j:set var="inStage" value="false"/>
                                     <j:if test="${help != null}">
-                                        <tr>
-                                            <td colspan="3"/>
+                                        <ttd:row>
+                                            <ttd:td colspan="3"/>
                                             <f:helpLink url="${help}"/>
-                                        </tr>
+                                        </ttd:row>
```

Comment:
From a quick test against 2.276 that @timja noticed in https://github.com/jenkinsci/workflow-cps-plugin/pull/411#pullrequestreview-578079700: With this change, clicking the help icon for the first directive (`agent`) works correctly, but clicking the icon for any other directive does nothing. I don't see any messages in my browser console when I click the help icons.

Interestingly, if I run this plugin using `master` against Jenkins 2.276, everything works correctly (and the appearance seems to be the same with or without this PR), so maybe we should just leave things as-is? Other than that, perhaps [`helpButtonOnClick` in `hudson-behavior.js`](https://github.com/jenkinsci/jenkins/blob/022a09cec0919e9bed7ce6535e13a19055b4190b/war/src/main/webapp/scripts/hudson-behavior.js#L765-L794) needs to be modified in some way?

Reply:
Oh, also I think the `colspan` attribute in `<ttd:td colspan="3"/>` is ignored since it is not defined as an attribute in the Jelly tag, right?